code/modules/hydroponics/seed.dm

```diff
@@ -25,6 +25,7 @@
 	var/apply_color_to_mob = TRUE  // Do we color the mob to match the plant?
 	var/has_item_product           // Item products. (Eggy)
 	var/force_layer
+	var/harvest_sound = null		//Vorestation edit - sound the plant makes when harvested
```

Comment:
Given the chance, we should send these to Polaris with some sounds that aren't memes.